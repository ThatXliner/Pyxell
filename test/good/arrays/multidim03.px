
a = [for i in 0..1000 yield [['X'], ['Y'], ['Z']] * i]
print a[0].length, a[-1].length
print a[-4][-3][0], a[-3][-2][0], a[-2][-1][0]
