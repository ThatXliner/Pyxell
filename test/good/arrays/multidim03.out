0 3000
X Y Z
